Fix/hypophosphatasia page

jhossyd3v opened the pull request:
- Fix of component 'Highlight Information' on content and complement of footnote (from props to slots)
- Fix of styles for some containers, titles, smaller paragraphs, charts, sub and sup HTML tags

Commit:
feat: :construction: Update component highligh information

Change props of content and complement of footnote for slots, add styles for sub HTML tag and uppercase for titles

Commit:
style: :lipstick: Update styles for desktop

Adjust some styles of footnotes, sup HTML tags, chart images, containers, titles on Hypophosphatasia Page for desktop

HackMort reviewed (changes requested):
1. Please remove the dummy text at the start of the page and match the ids of the section to its corresponding nav, so the internal-nav works as expected.

2. The content should not be wrapped yet, it should go all the way to the edge of the screen, please create a backup of whay you have, so if they want this it woud be easy to switch.

3. Looks like we are missing some images:
![image](https://user-images.githubusercontent.com/6466646/233504208-b9dd493d-6896-4988-8647-90e2661de79a.png)
Also the font-size of these green section/headings are not matching the Figma.

4. Check font-sizes:
![image](https://user-images.githubusercontent.com/6466646/233503858-9f348315-37ee-441c-82de-fc95ad9ff87d.png)

5. This section does not look right:
![image](https://user-images.githubusercontent.com/6466646/233504038-1efc99d7-4ebc-4e4a-af45-8547d3fdd5ab.png)

Commit:
feat: :construction: Update internal navigation

Delete the texts with lorem ipsum to test internal navigation, add ids to sections according to annotations in Figma

Commit:
fix: :bug: Fix styles font-sizes and positions

Fix font-sizes of titles and paddings, margins or positions of some sections , update styles on some screen sizes

Commit:
style: :lipstick: Remove wrapper and fix styles

Remove wrapper from sections on hypophosphatasia page, fix broken styles after remove wrapper

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into fix/hypophosphatasia-page

Commit:
fix: :bug: Change id for internal navigation

Change id of the section called disease to fix an issue with internal navigation

Commit:
style: :lipstick: Edit styles to extend containers with shapes

Fix styles for containers with shapes until the nearest border

Commit:
style: :construction: Change ids for internal navigation from titles to sections

Add ids for internal navigation to container sections instead of titles in sections

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into fix/hypophosphatasia-page

HackMort committed:
Change fonts from px to rems

HackMort reviewed (approved):
Looking pretty good!
